src/app/static/package.json

```diff
@@ -11,10 +11,10 @@
     "copy": "cp -r ./templates/*.ftl ./public && cp -r ./assets/* ./public",
     "test": "vitest --run",
     "browser-test": "npx playwright test",
-    "integration-test": "jest --testPathIgnorePatterns adr-dataset -c jest.integration.config.js",
-    "adr-integration-test": "jest --testPathPattern adr-dataset -c jest.integration.adr.config.js",
-    "integration-test-docker": "jest --testPathIgnorePatterns adr-dataset -c jest.integration.docker.config.js",
-    "adr-integration-test-docker": "jest --testPathPattern adr-dataset -c jest.integration.docker.adr.config.js",
+    "integration-test": "vitest --config ./test-integration-configs/integration.ts --run",
+    "adr-integration-test": "vitest --config ./test-integration-configs/adr-integration.ts --run",
+    "integration-test-docker": "vitest --config ./test-integration-configs/docker-integration.ts --run",
+    "adr-integration-test-docker": "vitest --config ./test-integration-configs/adr-docker-integration.ts --run",
```

Comment:
Do you remember why we run some of these under docker and other not? I can't remember why the 2 configurations here

Reply:
i think it was just to test that they can connect via docker i guess, as the code is run in docker containers, i think a future smoke test might be sufficient though if we even want to test that, the only things that are different is the connection url and the fact that we start the server in a docker container i think

Reply:
I think the feeling was possibly that the integration tests ought to run in docker on CI to reflect a real deployment, but we wanted to keep option to run on metal for local dev. Suspect there might have been at least one gotcha where tests passed on metal but there was an issue in docker which only showed up in manual testing on deployed instance. 
Not sure we should ditch them tbh, they don't really cost us anything. Of course if we had a fuller e2e test suite...